src/components/Container/src/Container.vue

```diff
@@ -10,22 +10,29 @@
 	>
 		<header :class="$s.Header">
 			<div
-				v-if="label"
+				v-if="hasLabel"
 				:class="$s.Label"
 			>
-				{{ label }}
-
+				<!-- @slot container label -->
+				<slot name="label">
```

Comment:
Ooh nice, using the prop value as the default slot content. 👌

Reply:
Yup, a tip I learned from Orbit :)